android/app/src/main/java/com/example/makeat/ui/LoginPage.kt

```diff
@@ -78,7 +79,7 @@ fun LoginPage(modifier: Modifier = Modifier) {
                 Text("I don't have an account", style = MaterialTheme.typography.bodyMedium)
             }
 
-            MButtonPrimary(text = "Enter", onClick = { TODO() /* Action login */ })
+            MButtonPrimary(text = "Enter", onClick = { onLoginSuccess() } )
```

Comment:
Tu peux peut-être faire
```suggestion
            MButtonPrimary(text = "Enter", onClick = onLoginSuccess )
```

Reply:
Oui, ici, j'aurai le même comportement, mais uniquement parce que pour le moment, il n'y a aucune logique de login avec requêtes à l'API, etc. C'est pour ça que je le laisse comme ça pour le moment.